README.md

```diff
@@ -43,10 +43,12 @@ jobs:
   serial: true
   plan:
   - get: resource-web-app
+  - task: build
+    file: resource-web-app/build.yml
   - put: resource-deploy-web-app
     params:
-      manifest: resource-web-app/manifest.yml
-      path: resource-web-app
```

Comment:
So copy the whole app directory? That definitely works but I had actually thought you guys would argue against that haha. I was thinking that you were encouraging just extracting the important parts of a task's working directory. I'm happy for this example to work either way but I think with apps that compile to a binary the outputs feature makes it feel cleaner to just copy the binary IMO.